whitebox/schemas/inferenceRow.py

```diff
@@ -13,9 +13,13 @@ class InferenceRowBase(BaseModel):
     actual: Optional[float]
 
 
-class InferenceRow(InferenceRowBase, ItemBase):
+class InferenceRowCreateDto(InferenceRowBase):
     pass
 
 
-class InferenceRowCreateDto(InferenceRowBase):
+class InferenceRowPreDb(InferenceRowBase):
+    is_used: bool
```

Comment:
Also @sinnec I tried to add some rows with the same date, testing the true/false flag and worked fine.